### PR TITLE
feat(stdlib)!: Move Power to runtime and add Operator

### PR DESCRIPTION
**Pull request by spotandjake**
This pr moves power to the runtime and adds pervasives as an operator. I didnt see anywhere to test the operator or functions in the runtime but power is well tested in the number test suite.

Closes: #1476

### CODE REVIEW

File: stdlib/pervasives.md [hunk 1]
```diff
@@ -532,6 +532,32 @@ Returns:
 |----|-----------|
 |`Number`|The modulus of its operands|
 
+### Pervasives.**(**)**
```
Comment (phated) [inline stdlib/pervasives.md:535]: @spotandjake I think you found a bug with graindoc. Since `*` is bolding, this isn't shown correctly.

Comment (spotandjake) [reply stdlib/pervasives.md:535]: f that is a good point, I guess ideally characters should be escaped.

**Comment by phated** [comment]
Blocked by https://github.com/grain-lang/grain/issues/1550

File: stdlib/pervasives.md (hunk 1 above)
Comment (phated) [reply stdlib/pervasives.md:512]: This was fixed by #1685 so this PR is unblocked now.

**Pull request closed without merging**